src/pages/CreateCirclePage/index.tsx

```diff
@@ -0,0 +1,2 @@
+import CreateCirclePage from './CreateCirclePage';
```

Comment:
i'm just bikeshedding now, but this boilerplate could be skipped if you just had `pages/CreateCirclePage.tsx` (or less redundantly, `pages/CreateCircle.tsx`)

Reply:
I think that having folders is good to be consistent when there are more large pages

Reply:
It could be done like the components